chapters/annotations.tex

```diff
@@ -1777,16 +1776,17 @@ \subsubsection{Conversion rules}\label{conversion-rules}
   number = 10, use_numberPort = not true);
 Modelica.Blocks.Math.LinearDependency linearDep(y0 = 2, k1 = 0, k2 = 2);
 model A
-  import Modelica.Electrical.Machines.BasicMachines;
-  extends BasicMachines.QuasiStaticDCMachines.DC_ElectricalExcited;
+  import My.Library;
+  extends Library.Package.ActualClass;
 end A;
 model B
   extends A;
-  Boolean b = a.quasiStatic;
+  Boolean b = correct;
```

Comment:
Could you just explain what happened to the `a` here?  What is the corresponding change in the conversions?

Reply:
It was an error before.

Reply:
Good!  Resolving.